lib/stdlib/src/json.hrl

```diff
@@ -0,0 +1,144 @@
+
+-define(is_ascii_escape(Byte),
```

Comment:
Wouldn't checking the range and just the two outliers be more performant and easier to read ?

Reply:
No, this implementation is significantly more performant. The compiler & JIT generate an efficient jump table in this case that is also fused with subsequent calls, e.g. to `escape/1`. Range comparisons do a lot of repeated work.